index.bs

```diff
@@ -345,7 +345,7 @@ events are also not excluding inputs.
 <pre class="idl">
   [Exposed=Window]
   interface LayoutShift : PerformanceEntry {
-    readonly attribute long value;
```

Comment:
Should this be `double` or `unrestricted double`? The latter allows for NaN and others, but not sure if you need them or not.

Reply:
I can't think of any reason we would need infinite or NaN values here.